migrations/sqls/20201028204537-followups-up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS followUps (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  serviceId INT NOT NULL,
```

Comment:
As I looked this over, it crossed my mind that maybe a call could discuss multiple CU services? I guess we could have them create two follow ups in that case?

Reply:
Multiple CU services would be useful. Would it be incoming as an array of service ids in the json object? Might be something to ask Vicky or Mayra @joeldenning

Reply:
To implement multiple invoices, a separate table would be best - one that has the follow up id and the service id. A "join table." Maybe called `followUpServices`